src/commands/report-failure.ts

```diff
@@ -58,7 +57,7 @@ export default async function reportFailure({ env }: MainArgs): Promise<void> {
   const issueTitle = getIssueTitle(runId);
   // https://help.github.com/en/github/searching-for-information-on-github/searching-issues-and-pull-requests
   const issueSearch = await github.search.issuesAndPullRequests({
-    q: `repo:${owner}/${repo} is:issue label:CI in:title ${issueTitle}`,
+    q: `repo:${owner}/${repo} state:open is:issue label:CI in:title ${issueTitle}`,
```

Comment:
Just calling this out as a bug fix. I kept this in the same PR because without the tests it is impossible to actually hit the issue (or prove that it is fixed).